### PR TITLE
Add ccache

### PR DESCRIPTION
**Pull request by satmandu**
Adds ccache

Works properly:
- [x] x86_64
- [x] i686
- [x] armv7l

### CODE REVIEW

File: packages/ccache.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  })
+  binary_sha256 ({
+     aarch64: '525bce425116fadb8ffaf352df87bde4f424380a1d4a98915d52de005aa0663d',
+      armv7l: '525bce425116fadb8ffaf352df87bde4f424380a1d4a98915d52de005aa0663d',
+        i686: '14ebe59db9eb33c86f867153bda9340fa5732a2ad19f48fe0e8900f238b43015',
+      x86_64: '5c4c9e014f23977f2f031eb9f9b367881a20fa8effeac9e38289cd723aa11e62',
+  })
+
+
+  depends_on 'zlibpkg'
```
Comment (uberhacker) [inline packages/ccache.rb:25]: This is a core package so no need to list here.

File: packages/ccache.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/ccache-4.1-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/ccache-4.1-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/ccache-4.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     aarch64: '525bce425116fadb8ffaf352df87bde4f424380a1d4a98915d52de005aa0663d',
+      armv7l: '525bce425116fadb8ffaf352df87bde4f424380a1d4a98915d52de005aa0663d',
+        i686: '14ebe59db9eb33c86f867153bda9340fa5732a2ad19f48fe0e8900f238b43015',
+      x86_64: '5c4c9e014f23977f2f031eb9f9b367881a20fa8effeac9e38289cd723aa11e62',
+  })
```
Comment (uberhacker) [inline packages/ccache.rb:22]: Looks like you figured it out. :)

File: packages/ccache.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  Dir.mkdir 'build'
+    Dir.chdir 'build' do
+      system "cmake -G Ninja \
+      #{CREW_CMAKE_OPTIONS} \
+      -DCMAKE_INSTALL_SYSCONFDIR=#{CREW_PREFIX}/etc \
+      -DZSTD_FROM_INTERNET=ON \
+      .."
+      system "ninja"
+    end
+  end
```
Comment (uberhacker) [inline packages/ccache.rb:40]: Extra `end`?  No need to terminate the `Dir.mkdir 'build'` line.

File: packages/ccache.rb (hunk 2 above)
Comment (satmandu) [reply packages/ccache.rb:22]: yup!

File: packages/ccache.rb (hunk 3 above)
Comment (uberhacker) [reply packages/ccache.rb:39]: Actually, nevermind.  Please indent lines 29-31 by 2 spaces.

File: packages/ccache.rb [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  
+  def self.install
+  system "DESTDIR=#{CREW_DEST_DIR} ninja -C build install"
```
Comment (uberhacker) [inline packages/ccache.rb:42]: Can you please indent this line?

File: packages/ccache.rb (hunk 3 above)
Comment (satmandu) [reply packages/ccache.rb:39]: Sure

File: packages/ccache.rb (hunk 4 above)
Comment (satmandu) [reply packages/ccache.rb]: done

**Pull request merged**